backend/development.ini.sample

```diff
@@ -870,7 +870,12 @@ caldav.radicale.headers.Access-Control-Expose-Headers = Etag
 ## Jitsi Meet configuration parameters
 ; call.jitsi_meet.url =
 # example:
-; call.jitsi_meet.url = https://meet.jit.si
+# call.jitsi_meet.url = https://meet.jit.si
+
+## Time (in ms) before the call is considered unanswered
+; call.unanswered_timeout =
+# example:
+# call.unanswered_timeout = 30000
```

Comment:
Its better to give an example with not default value. 
~~~
# Example: timeout after 60s
# call.unanswered_timeout = 60000
~~~

Reply:
Ok, I'll change the example.